src/main/java/edu/ksu/canvas/oauth/OauthTokenRefresher.java

```diff
@@ -33,39 +35,42 @@ public OauthTokenRefresher(String clientId, String clientSecret, String canvasUr
 
     public TokenRefreshResponse getNewToken(String refreshToken) throws IOException {
         LOG.debug("Getting a fresh OAuth access token");
-        RequestConfig config = RequestConfig.custom()
-                .setConnectTimeout(TIMEOUT_SECONDS*1000)
-                .setSocketTimeout(TIMEOUT_SECONDS*1000)
-                .build();
+        ConnectionConfig connConfig = ConnectionConfig.custom()
+            .setConnectTimeout(TIMEOUT_SECONDS*1000, TimeUnit.MILLISECONDS)
```

Comment:
I wonder if using `Duration` would be better (every time I see multiplication it always makes me want to find a constant.)

Since we're changing it anyway...

I think that we can do this:

`.setConnectionTimeout(Timeout.of(Duration.ofSeconds(TIMEOUT_SECONDS)))`

Reply:
Updated in https://github.com/UCBoulder/canvas-api/pull/4/commits/42048da9b544dc1492b4b7717802907df6e6ce2d